modules/ROOT/pages/6.6.1-release-notes.adoc

```diff
@@ -102,9 +102,43 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.6.1 also includes the following addition:
 
-=== New `pad_empty_with_br` option that can be set to `true` to pad empty block elements with `<br>` tag instead of nbsp character
+=== New filtering option: `pad_empty_with_br`
 //#TINY-9861
 
+{productname} 6.6.1 includes a further content filtering option, `pad_empty_with_br`.
+
+Content filtering options change the way {productname} manages input and output.
+
+The `pad_empty_with_br` option, when set to `true`, places a `+<br>+` tag within empty block elements rather than the default non-breaking space entity, `+&nbsp;+`.
+
+That is, when a {productname} instance includes the following setting
+
+[source,js]
+----
+pad_empty_with_br: true,
+----
+ 
+this input
+
+[source,html]
+----
+<p>&nbsp;</p>
+----
+
+is automatically stored and rendered as
+
+[source,html]
+----
+<p><br></p>
+----
+
+[NOTE]
+====
+An equivalent filtering option, `padd_empty_with_br` was xref:migration-from-5x.adoc#previously-deprecated-items-now-removed[removed from {productname} 6.0.0].
```

Comment:
@brian-forte was this meant to be `padd_empty_with_br` or was this a typo with the extra `d`?

```suggestion
An equivalent filtering option, `pad_empty_with_br` was xref:migration-from-5x.adoc#previously-deprecated-items-now-removed[removed from {productname} 6.0.0].
```

Reply:
> was this meant to be padd_empty_with_br or was this a typo with the extra d?

The name of the original, undocumented, option had the double-d. That is, the option was live with a typo in its name.